en-US/O.xml

```diff
@@ -83,6 +83,16 @@
 				</para>
 			</listitem>
 
+		</varlistentry>
+		<varlistentry id="once">
+		 <term>once</term>
+			<listitem>
+			 <para>
+				 Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”
```

Comment:
Avoid "smart quotes." Not a big deal here, but we don't use them in courses and it's a good habit to get into to not use them at all.

Reply:
Made requested change.